range_queries/sparse_table.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file sparse_table.cpp
+ * @brief Implementation of [Sparse Table](https://en.wikipedia.org/wiki/Range_minimum_query) data structure
+ *
+ * @details
+ * Sparse Table is a data structure, that allows answering range queries.
+ * It can answer most range queries in O(logn), but its true power is answering range minimum queries
+ * or equivalent range maximum queries). For those queries it can compute the answer in O(1) time.
+ *
+ * Running Time Complexity \n
+ * Build : O(NlogN) \n
+ * Range Query : O(1) \n
+*/
+
+#include <vector>
+#include <cassert>
+#include <iostream>
+#include <algorithm>
+
+/**
+ * @namespace range_queries
+ * @brief Range Queries algorithms
+ */
+namespace SparseTable {
```

Comment:
This is incorrect.
```suggestion
namespace range_queries {
```
You may want to add a namespace member.
Confused? See #962 as a reference.